python/lsst/pex/config/config.py

```diff
@@ -1010,9 +1010,17 @@ def items(self):
         return self._storage.items()
 
     def __contains__(self, name):
-        """!Return True if the specified field exists in this config.
+        """Return `True` if the specified field exists in this config.
 
-        @param[in] name  field name to test for
+        Parameters
+        ----------
+        name : `str`
+            Field name to test for.
+
+        Returns
+        -------
+        in : `bool`
+            `True` if the specified field exists in the config.
```

Comment:
I've never been sure whether docstrings for special methods like this actually get rendered. I guess this does?

Reply:
This one I fixed because it was the only old style docstring in the code.